components/features.js

```diff
@@ -0,0 +1,99 @@
+//Feature Component
```

Comment:
Really awesome job using a constructor with methods here, love you added sound to the icons in your footer!